l10n_th_withholding_tax_cert/models/account_move.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2019 Ecosoft Co., Ltd (https://ecosoft.co.th/)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from odoo import _, fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    wt_cert_ids = fields.One2many(
+        comodel_name="withholding.tax.cert",
+        inverse_name="move_id",
+        string="Withholding Tax Cert.",
+        readonly=True,
+    )
+    wt_cert_cancel = fields.Boolean(compute="_compute_wt_cert_cancel")
```

Comment:
I am not cleared, what is the meaning of this field? I saw it like this,
```
    payment_id = fields.Many2one(
        comodel_name="account.payment",
        string="Payment",
        copy=False,
        readonly=True,
        states={"draft": [("readonly", False)]},
        domain="[('partner_id', '=', supplier_partner_id),"
        "('wt_cert_cancel', '=', True)]",
        ondelete="restrict",
        tracking=True,
    )
```
So you want to select only document (i..e, payment) that has some cert that is not cancelled???
Even so, I think following function is not easy to understand. Can you explain in a simpler way of `wt_cert_cancel` ?